en/orm/query-builder.rst

```diff
@@ -708,7 +708,7 @@ methods being combined with ``AND``. The resulting SQL would look like:
 .. deprecated:: 3.5.0
     As of 3.5.0 the ``orWhere()`` method is deprecated. This method creates
     hard to predict SQL based on the current query state.
-    Use ``where()`` instead as it has more predictable and easier
+    Use ``where()`` (see: https://api.cakephp.org/3.8/source-class-Cake.Database.Query.html#758-886) instead as it has more predictable and easier
```

Comment:
Linking to the source seems a bit much. We could link to https://api.cakephp.org/3.8/class-Cake.Database.Query.html#_where instead.

Reply:
sounds good enough to me ;)